santoriniWebsite/client/package.json

```diff
@@ -42,5 +42,8 @@
       "last 1 firefox version",
       "last 1 safari version"
     ]
+  },
+  "devDependencies": {
+    "@babel/plugin-transform-private-property-in-object": "^7.25.7"
```

Comment:
this doesn't seem used and it's also a duplicate key

Reply:
actually it seems to be subdependency?

Reply:
Oh, I haven't committed my latest changes yet and I upgraded back to version 5 and deleted the Babel subdependency . These two changes was bc to troubleshoot errors at the time. 
I did figure out one key change to make the website work tho,
 "scripts": {
    "start": "set NODE_OPTIONS=--openssl-legacy-provider && react-scripts start",
    ...
    }
    
    Basically, this flag solved the issue with the start command and made it compile.